tests/terraform/graph/checks_infra/attribute_solvers/jsonpath_equals_solver/test_solver.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from tests.terraform.graph.checks_infra.test_base import TestBaseSolver
+
+TEST_DIRNAME = os.path.dirname(os.path.realpath(__file__))
+
+
+class TesJsonpathEqualsSolver(TestBaseSolver):
+    def setUp(self):
+        self.checks_dir = TEST_DIRNAME
+        super(TesJsonpathEqualsSolver, self).setUp()
```

Comment:
```suggestion
        super().setUp()
```
no need to set it here, it will automatically the parent classes `setUp`

Reply:
fixed